### PR TITLE
Tsk 4597

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
@JohannesDoberer - Hey Hannes, just a reminder not to merge this pull request before Flyspecks has done their review.

File: docu/files/ExtModules.html.md [hunk 1]
```diff
@@ -837,16 +836,66 @@ Use the context variable `currentLanguage` to determine the language to which th
     </tr>
 </table>
 
-Based on the information that this variable provides, developers select localized content that they have previously configured using the technology of their choice.
+Use the context variable `currentLanguage` to determine the current language set for the Builder module. Based on the information that this variable provides, developers select localized content that they have previously configured using the technology of their choice.<br><br>
```
Comment (MindyJamiel) [inline docu/files/ExtModules.html.md:839]: Throughout, format variable and parameter names (as opposed to their values) in bold, not code format: **currentLanguage**. The value of **currentLanguage** would be in code format: `EN` or `DE`.

File: docu/files/ExtModules.html.md [hunk 2]
```diff
@@ -837,16 +836,66 @@ Use the context variable `currentLanguage` to determine the language to which th
...
+```
+<br>
+<b>2. Include the directive in the view that binds to the controller. In this case, `ng-controller`.</b>
+
+```
+<div class="panel-body" ng-controller="settingsCtrl">
+
+```
+<br>
+<b>3. Create a JSON file for each language that you would like to translate.</b> In this example, there is a file for German and English.  The file containing German content has a name such as locale_de.json with this notation:
```
Comment (MindyJamiel) [inline docu/files/ExtModules.html.md]: Format file names in bold: *locale_de.json**

**Review by MindyJamiel — CHANGES_REQUESTED** [review]
Looks great; approved pending minor formatting updates mentioned in the comments.

**Pull request merged**